src/components/DataContainer/DataContainer.tsx

```diff
@@ -0,0 +1,26 @@
+import { getAllData } from "@utils";
+import React, { useEffect, useState } from "react";
+
+const URL = "http://localhost:8000/api/v1/";
+
+export default function DataContainer() {
```

Comment:
When merging the two templates' logic together, I put the CtD stuff in this component